src/_data.json

```diff
@@ -496,29 +496,44 @@
   ],
   "theteam": [
     {
-      "name": "Wolfram Kriesing",
-      "photo": "wolfram.jpg",
-      "email": "w@kriesing.de"
+      "name": "Agnes Köhler",
+      "photo": "agnes.png",
+      "email": "koehler.agnes@gmail.com"
+    },
+    {
+      "name": "David Losert",
+      "photo": "david.jpg",
+      "email": "david@david-losert.com"
+    },
+    {
+      "name": "Jörn Bernhardt",
+      "photo": "joern.jpg",
+      "email": "joern.bernhardt@compose.us"
+    },
+    {
+      "name": "Michael Zoidl",
+      "photo": "michael.png",
+      "email": "michael@entwicklerstube.com"
+    },
+    {
+      "name": "Patrick Müller",
+      "photo": "patrick.jpg",
+      "email": "mp@floatec.de"
     },
     {
       "name": "Robert Hostlowsky",
       "photo": "robert.png",
       "email": "robert.hostlowsky@codecentric.de"
     },
     {
-      "name": "Marco Emrich",
-      "photo": "marco.jpg",
-      "email": "marco.emrich@codecentric.de"
-    },
-    {
-      "name": "Marc Emmanuel",
-      "photo": "marc.jpg",
-      "email": "marc.emmanuel@virtual-identity.com"
+      "name": "Simon Hofmann",
+      "photo": "simon.png",
+      "email": "simon@s1h.org"
     },
     {
-      "name": "Sebastian Frerichs",
```

Comment:
Sebastian is part of theam in 2020, too.

Reply:
I am terribly sorry, I didn't know.
I added him again! :)